cpp_guidelines/.clang-format

```diff
@@ -0,0 +1,107 @@
+---
+Language: Cpp
+Standard: c++17
+DisableFormat:   false
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveBitFields: true
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: true
+AlignEscapedNewlines: Right
+AlignOperands: true
+AlignTrailingComments: true
+AllowAllArgumentsOnNextLine: false
+AllowAllConstructorInitializersOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: Never
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortEnumsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: None
+AllowShortIfStatementsOnASingleLine: Never
+AllowShortLambdasOnASingleLine: All
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterReturnType: None
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: Yes
+BinPackArguments: false
+BinPackParameters: false
+BitFieldColonSpacing: Both
+BreakBeforeBinaryOperators: All
+BreakBeforeBraces: Allman
+BreakBeforeTernaryOperators: true
+BreakConstructorInitializers: BeforeComma
+BreakInheritanceList: BeforeComma
+BreakStringLiterals: true
+ColumnLimit:     79
+CommentPragmas:  '^ COMMENT pragma:'
+CompactNamespaces: false
+ConstructorInitializerAllOnOneLineOrOnePerLine: true
+ConstructorInitializerIndentWidth: 4
+ContinuationIndentWidth: 4
+Cpp11BracedListStyle: true
+DeriveLineEnding: false
+DerivePointerAlignment: false
+ExperimentalAutoDetectBinPacking: false
+FixNamespaceComments: true
+IncludeBlocks:   Preserve
+IncludeCategories:
+  - Regex:           '^"(llvm|llvm-c|clang|clang-c)/'
+    Priority:        2
+    SortPriority:    0
+  - Regex:           '^(<|"(gtest|gmock|isl|json)/)'
+    Priority:        3
+    SortPriority:    0
+  - Regex:           '.*'
+    Priority:        1
+    SortPriority:    0
+IncludeIsMainRegex: '(Test)?$'
+IncludeIsMainSourceRegex: ''
+IndentCaseBlocks: true
+IndentCaseLabels: false
+IndentExternBlock: AfterExternBlock
+IndentGotoLabels: true
+IndentPPDirectives: AfterHash
+IndentWidth: 4
+IndentWrappedFunctionNames: false
+KeepEmptyLinesAtTheStartOfBlocks: false
+MacroBlockBegin: ''
+MacroBlockEnd:   ''
+MaxEmptyLinesToKeep: 2
+NamespaceIndentation: All
+OperandAlignmentStyle: Align
+PenaltyBreakAssignment: 2
+PenaltyBreakBeforeFirstCallParameter: 19
+PenaltyBreakComment: 300
+PenaltyBreakFirstLessLess: 120
+PenaltyBreakString: 1000
+PenaltyBreakTemplateDeclaration: 10
+PenaltyExcessCharacter: 1000000
+PenaltyReturnTypeOnItsOwnLine: 1000
+PointerAlignment: Left
+ReflowComments: true
+SortIncludes: true
+SortUsingDeclarations: true
+SpaceAfterCStyleCast: true
+SpaceAfterLogicalNot: false
+SpaceAfterTemplateKeyword: true
```

Comment:
**VOTE**
```
current | false       |
--------|-------------|
 0      |   1         |
```

Vote to change to `false`. IMO space after the template keyword looks strange.

# Example:
current setting:
```C++
template <typename T_MplSeq, typename T_TmpResult = bmpl::vector0<>,
          bool T_isEmpty = bmpl::empty<T_MplSeq>::value>
struct AllCombinations;    
```

Suggestion:
```C++
template<typename T_MplSeq, typename T_TmpResult = bmpl::vector0<>,
         bool T_isEmpty = bmpl::empty<T_MplSeq>::value>
struct AllCombinations;
```

Reply:
current | false       |
--------|-------------|
1      |   1         |

Again, I would like to only give half a vote to current.

Reply:
current | false       |
--------|-------------|
 1      |   1         |

Reply:
current | false |
--------|-----------|
 2      |   2

Reply:
current | false
-- | --
2 | 3

Reply:
current | false       |
--------|-------------|
 2      |   4         |

Reply:
current | false |
--------|-----------|
 3      |   4

Reply:
current | false
-- | --
4 | 4

Reply:
Reminder: Voting will be closed on 2020-09-26 at 11:42 HZDR time.

Reply:
Voting closed but we have a tie. Does one of the voters want to change their vote?

Reply:
Let me help:
current | false
-- | --
4 | 5

Reply:
Voting closed, the winner is `false`.